packages/editor/src/client/atoms/dateTimePicker.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react'
+import {DatePicker, FormGroup, ControlLabel, Form} from 'rsuite'
+import {useTranslation} from 'react-i18next'
+
+export interface DateTimeProps {
+  dateTime?: Date
+  label: string
+  changeDate(publishDate?: Date): void
+}
+
+export function DateTimePicker({dateTime, label, changeDate}: DateTimeProps) {
+  const {t} = useTranslation()
+
+  return (
+    <Form fluid={true}>
+      <FormGroup>
+        <ControlLabel>{label}</ControlLabel>
+        <DatePicker
+          style={{marginRight: 8}}
+          placement="auto"
+          value={dateTime?.getTime() === dateTime?.getTime() ? dateTime : undefined}
```

Comment:
i don't understand what you do here?

Reply:
This is an attempt to only display the date/time if it's a valid date. It prevents an invalid date getting set when user clears the picker, but it's not working great.

Reply:
@penk85 you mean you are checking if it's a valid date? I suggest we have a util method and use it here
like:
`function isValidDate(d) {
  return d instanceof Date && !isNaN(d);
}
`
and use it like `value={isValidDate(dateTime) ?  dateTime : undefined}` 

But I think that rsuite already handling errors like these?

Reply:
@bishoymelek the way the component is set up, it sets only time or date in the value which creates an error in the form. We opted to disable clearing the form since anyhow there's no case here for an optional date.